src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -448,4 +449,35 @@ protected function getContextShopId()
     {
         return $this->getContext()->shop->id;
     }
+
+    /**
+     * Get error by exception from given messages
+     *
+     * @param Exception $e
+     * @param array $messages
+     *
+     * @return string
+     */
+    protected function getErrorMessageForException(Exception $e, array $messages)
```

Comment:
Shall we (in another PR) modify all existing controllers to reuse this one ?

Reply:
i guess we could.

Reply:
I put myself a reminder so we can merge the PR and not forget: https://github.com/PrestaShop/PrestaShop/issues/10580